### DISCUSSION TITLE
quarto render issues

### DISCUSSION THREAD

**Discussion by ericvmai** [quarto-dev/quarto-cli#2395] [Q&A]
I keep encountering differing versions of the same issue after I try render a quarto html project.

Error in normalizePath(path.expand(path), winslash, mustWork) :
  path[1]="site_libs/clipboard-0.0.1": The system cannot find the file specified
Calls: .main ... html_dependencies_as_string -> lapply -> FUN -> normalizePath

Can anyone explain to me exactly what is going on here? This happens after I tell it to render to html. This message comes up in the terminal after running through everything.

**Comment by dragonstyle** [comment]
It looks like we're having some kind of issue resolving a path to a dependency that Quarto is trying to include in the document output. Do you mind sharing a sample document that causes the issue and perhaps anything about the path or file system where the document resides that could be at all related?

**Reply by ericvmai** [reply]
Sorry I'm kind of new to posting in GitHub discussions. Below is my branch that I'm trying to test out my transition from bookdown to quarto.  The file path resides in a Google Drive and not on my local machine. Could that possibly be an issue?

https://github.com/hdadvisors/chesterfield-market-analysis/tree/quarto-transition

**Reply by mcanouil** [reply]
@ericvmai In GitHub Issues/Discussions, you can use markdown exactly the same way as in Quarto or Rmarkdown to format your message 😉

**Reply by dragonstyle** [reply]
> The file path resides in a Google Drive and not on my local machine. Could that possibly be an issue?

We have definitely seen some strangeness causes by google drive folders, though we haven't yet gotten to the bottom of why that is happening. Would you be able to try rendering not a google drive folder just to confirm that this is the issue?

**Reply by ericvmai** [reply]
that seems to have been the fix! However, should a browser open upon finishing an html render? Or am I mis-remembering?

**Reply by mcanouil** [reply]
It depends on how you did the rendering. From Quarto CLI `quarto render` or `quarto preview` with/without `--no-browser`, etc. (RStudio and VSCode are using those with some default parameters/options, _e.g._, the "render" button in VSCode execute `quarto preview index.qmd --no-browser --no-watch-inputs`).
